src/CSET/operators/regrid.py

```diff
@@ -202,6 +202,7 @@ def regrid_to_single_point(
     cube: iris.cube.Cube,
     lat_pt: float,
     lon_pt: float,
+    latlon_in_type: str,
```

Comment:
Do we want to set the default of this to `"rotated"` so existing code continues to work?